infrastructure/hosts/catcolab-next/default.nix

```diff
@@ -10,47 +12,49 @@ let
 in
 {
   imports = [
-    ../../modules/backend.nix
-    ../../modules/host.nix
-    ../../modules/backup.nix
-    "${inputs.nixpkgs}/nixos/modules/virtualisation/amazon-image.nix"
```

Comment:
Referencing `inputs` like this caused an infinite recursion error for very nixy reasons. `modulesPath` is the more "correct" method for this.